### PR TITLE
Add Scoop installation methods

### PR DESCRIPTION
**Pull request by Zliced13**
### Motivation:

I wanted to add scoop installation methods for swift, since I am the one who added it to [scoop](scoop.sh).

### Modifications:

I added scoop installation methods to these files, **_installing.md** and the **_windows.md**

### Result:

**_installing.md** and **_windows.md** will now have sections that detail how to install the dependencies and the Swift programming language using scoop.

### CODE REVIEW

**Comment by rashil2000** [comment]
The `scoop bucket add main` command can be omitted everywhere (because Main bucket is added by default during installation)

**Comment by KrisSchneider** [comment]
Should I just remove the part where it says `scoop bucket add main`?

**Comment by rashil2000** [comment]
Yes

**Comment by tomerd** [comment]
cc @compnerd @stevapple

File: download/_windows.md [hunk 1]
```diff
@@ -44,6 +44,37 @@ The [Windows Package Manager](https://docs.microsoft.com/windows/package-manager
    pip install six
    ~~~
    
+##### Install using Scoop
+
+Scoop can be installed into your system using these commands:
```
Comment (compnerd) [inline download/_windows.md:49]: Should this be "Scoop" or should it be `scoop`?  I don't think I have a strong opinion, both are referring to different aspects of the tool.

File: download/_windows.md [hunk 2]
```diff
@@ -44,6 +44,37 @@ The [Windows Package Manager](https://docs.microsoft.com/windows/package-manager
...
+   ~~~ pwsh
+   Set-ExecutionPolicy RemoteSigned -Scope CurrentUser # Optional: Needed to run a remote script the first time
+   irm get.scoop.sh | iex
```
Comment (compnerd) [inline download/_windows.md:52]: I think rewriting the command would be nice.

```suggestion
   Invoke-RestMethod -Url get.scoop.sh | Invoke-Expression
```

While more verbose, it is for copy-paste consumption, in which case, using this variation means that those unfamiliar with the aliases can tell at a glance what is happening.

File: getting-started/_installing.md [hunk 3]
```diff
@@ -93,6 +93,37 @@ The [Windows Package Manager](https://docs.microsoft.com/windows/package-manager
    pip install six
    ~~~
    
+##### Install using Scoop
+
+Scoop can be installed into your system using these commands:
+   ~~~ pwsh
+   Set-ExecutionPolicy RemoteSigned -Scope CurrentUser # Optional: Needed to run a remote script the first time
+   irm get.scoop.sh | iex
```
Comment (compnerd) [inline getting-started/_installing.md:101]: Similar

File: download/_windows.md (hunk 2 above)
Comment (KrisSchneider) [reply download/_windows.md:52]: sure that can work

File: getting-started/_installing.md (hunk 3 above)
Comment (compnerd) [inline getting-started/_installing.md:101]:
```suggestion
   Invoke-RestMethod -Url get.scoop.sh | Invoke-Expression
```

File: download/_windows.md (hunk 1 above)
Comment (KrisSchneider) [reply download/_windows.md:49]: I think Scoop would fit better

File: getting-started/_installing.md (hunk 3 above)
Comment (KrisSchneider) [reply getting-started/_installing.md:101]: Sure

File: download/_windows.md (hunk 1 above)
Comment (stevapple) [reply download/_windows.md:49]: +1 for "Scoop"

**Review by stevapple — COMMENTED** [review]
I’m concerned about the layout, since the change:
- skips H4 and jumps directly from H3 to H5;
- includes "Install required dependencies" for `winget` and `scoop` separately, but shares the "Install Swift" step.

How can we organize more properly here? Maybe by restructuring the instructions into three sections: VS, dependent tools and Swift?

**Comment by KrisSchneider** [comment]
@stevapple What do you mean by this:

> * skips H4 and jumps directly from H3 to H5;

**Comment by stevapple** [comment]
25380811-2EEA-4E0C-A4ED-5CF9C6A2A1B2

The structure looks weird in `download/_windows.md` and it also diverges from `getting-started/_installing.md`. We’d better keep the two identical.

In the long run one of these has to be eliminated because maintaining two separate copies is troublesome, pre requested by #4.

**Review by compnerd — APPROVED** [review]
LGTM; thanks!

File: download/_windows.md [hunk 4]
```diff
@@ -44,6 +44,37 @@ The [Windows Package Manager](https://docs.microsoft.com/windows/package-manager
...
+   ~~~
+   Start up a new Command Prompt and install the Python library six.
+   
+   ~~~ cmd
+   pip install six
+   ~~~
+
 0. Install Swift:
 
    Swift can be installed through the official installer directly, or using the Windows Package Manager as well.  Notice that Windows Package Manager release may be behind the official release.
```
Comment (stevapple) [inline download/_windows.md:80]: Should add Scoop here too.

File: getting-started/_installing.md [hunk 5]
```diff
@@ -93,6 +93,37 @@ The [Windows Package Manager](https://docs.microsoft.com/windows/package-manager
...
+   ~~~
+   Start up a new Command Prompt and install the Python library six.
+
+   ~~~ cmd
+   pip install six
+   ~~~
+
 0. Install Swift:
 
    Swift can be installed through the official installer directly, or using the Windows Package Manager as well.  Notice that Windows Package Manager release may be behind the official release.
```
Comment (stevapple) [inline getting-started/_installing.md:129]: Same here

File: download/_windows.md (hunk 4 above)
Comment (KrisSchneider) [reply download/_windows.md]: Done, It's added now

File: getting-started/_installing.md (hunk 5 above)
Comment (KrisSchneider) [reply getting-started/_installing.md]: Done, It's added to this page as well.

**Comment by Zliced13** [comment]
Hello, is there any more changes you would like added to this PR? Or can this PR be approved?

**Comment by tomerd** [comment]
@compnerd any further feedback? or is this ready to be merged?

**Comment by compnerd** [comment]
@tomerd I believe that this is ready to merge.  There are some small things that would be nice (i.e. can't scoop also run behind official releases?).  But, this looks good if the scoop definitions are present.  Getting it easier to install in different environments is definitely exciting.

**Comment by Zliced13** [comment]
> @tomerd I believe that this is ready to merge. There are some small things that would be nice (i.e. can't scoop also run behind official releases?). But, this looks good if the scoop definitions are present. Getting it easier to install in different environments is definitely exciting.

Scoop always auto-updates the packages to the latest version. You don't have to worry about it falling behind.

**Comment by Zliced13** [comment]
What conflicts do I have to resolve?

**Comment by kaishin** [comment]
> What conflicts do I have to resolve?

@Zliced13 You can see and fix them [here](https://github.com/apple/swift-org-website/pull/157/conflicts).

**Comment by Zliced13** [comment]
@kaishin I'm going to open another PR because I think this PR was a disaster.

**Pull request closed without merging**